index.js

```diff
@@ -215,7 +201,11 @@ function ucfirst(str) {
 exports.addTasks = (gulp, libraryName, srcGlob, webpackConfig, dtsGlob, options = {}) => { //eslint-disable-line max-params
     const libraryClassName = _.flow(_.camelCase, ucfirst)(libraryName);
 
-    systemjsBundle(gulp, { distName: libraryName, modules: options.modules, webpackConfig });
+    systemjsBundle(
```

Comment:
Pass the stream and the webpack as parameters in the `options` object

`
systemjsBundle(
        gulp,
        {
            distName: libraryName,
            modules: options.modules,
            webpackConfig: webpackConfig.systemjs,
            webpackStream,
            webpack
        }
    );
`